CHANGELOG.md

```diff
@@ -34,6 +36,18 @@ librdkafka v2.4.0 is a feature release:
    depending on the application logic (#4583).
 
 
+## Early Access
+ * The new consumer group rebalance protocol, defined in KIP 848, is still _not production-ready_ and _not supported_.
+   It's possible to try it in a non-production enviroment.
+
+   With this protocol the role of the Group Leader (a member) is removed and
+   the assignment is calculated by the Group Coordinator (a broker) and sent
+   to each member through heartbeats.
```

Comment:
I think these two paragraph can be swapped. We want to explain about the protocol first and then tell about the caveats.